### DISCUSSION TITLE
Revert skroutz homepage to its pre-bloated setup

### DISCUSSION THREAD

**Discussion by ghost** [keybraker/reSkroutzed#12] [Ideas]
Hello again, everyone!

![image](https://user-images.githubusercontent.com/68644945/236228199-fe6669c5-a1a4-48d5-aecf-903336ee92eb.png)

You may or may not have noticed that, very recently, the Skroutz homepage started being full of bloat. I haven't tested anything past gtopsis's latest commit but I assume we haven't patched this yet.

I'll try to investigate the issue and see if it's feasible to revert the homepage to its pre-ad era without a lot of hassle.

**Comment by keybraker** [comment]
It is feasible, but I am not sure this is something we should alter. We should probably consider creating a new extension that is called **Skroutz-Debloat** or **Skroutz-Purify** or something of that sort. The best name would be Skroutz-Vanced, but this is already taken, unfortunately.

**Reply by ghost** [reply]
Hmm, technically though, it *is* primarily ads and/or sponsored content

**Reply by keybraker** [reply]
I have thought about it once more, and think it should be handled as a promotional video.
Decreasing the opacity of the entire div, but not removing it.

**Comment by keybraker** [comment]
@ouranya I have pivoted reSkroutzed, towards a more inclusive extension to fix bad practices, enhance experience and remove ads and bloat added by skroutz.
This has now been implemented, and things that fall into the same category get a new option to remove them if wanted.
